poetry/masonry/utils/tags.py

```diff
@@ -40,11 +40,7 @@ def get_impl_ver(env):
 def get_impl_version_info(env):
```

Comment:
It might be possible to replace this module by `packaging.tags` (which supports passing information about the currently running interpreter via arguments).